internal/namespaces/init/init.go

```diff
@@ -231,6 +233,24 @@ func initCommand() *core.Command {
 				args.InstallAutocomplete = scw.BoolPtr(installAutocomplete)
 			}
 
+			// Ask whether to remove v1 configuration file is it exists
+			homeDir, err := os.UserHomeDir()
```

Comment:
Should we make `getConfigV1FilePath` in SDK public ?

Reply:
Not a big fan since it would encourage to use it